addons/project_issue/views/project_project_views.xml

```diff
@@ -24,7 +24,7 @@
         <field name="arch" type="xml">
             <field name="use_tasks" position="after">
                 <field name="use_issues"/>
-                <field name="issue_ids" invisible="1"/>
```

Comment:
Don't remove this field as other modules can expect to have it there for extending the view.  Does this impact in the performance?